invenio_records_files/config.py

```diff
@@ -24,14 +24,17 @@
     }
 
 * ``<*_REST_ENDPOINTS>`` corresponds to `Invenio-Records-REST endpoint
-  configurations names <https://invenio-records-rest.readthedocs.io/en/
-  latest/configuration.html#invenio_records_rest.config
-  .RECORDS_REST_ENDPOINTS>`_ you have defined in your application.
-
-* ``<endpoint-prefix>`` is the unique name of your endpoint as you can see in
-  `Invenio-Records-REST <https://invenio-records-rest.readthedocs.io/en/
-  latest/configuration.html#invenio_records_rest.
-  config.RECORDS_REST_ENDPOINTS>`_ like configuration.
+  configurations names
+  <https://invenio-records-rest.readthedocs.io/en/latest/configuration.html
+  #invenio_records_rest.config.RECORDS_REST_ENDPOINTS>`_
+  that you have defined in your application.
+
+* ``<endpoint-prefix>`` is the unique name of the endpoint configuration as it
+  is defined in `Invenio-Records-REST
+  <https://invenio-records-rest.readthedocs.io/en/latest/configuration.html
```

Comment:
do you think it is worth to mention that the `endpoint_prefix` has to match `pid_type`?